support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala

```diff
@@ -181,6 +181,49 @@ object DigiPackFailureHandlerManualTest extends Matchers {
         sys.exit(0)
       }
   }
+}
+
+object GuardianWeeklyFailureHandlerTest extends Matchers {
```

Comment:
Do you think there’s any unit tests to add as well, or do the existing ones still cover?

Reply:
@jfsoul Unit tests are not possible here unfortunately, these are manual checks only. How would you unit that that a specific email box has received an email without writing a mail server? I might be wrong but I don't think that can work.

Reply:
Sure, probably not - I was just thinking was there anything to add in `FailureHandlerSpec` class for example, but probably new product failure emails don't change failure handling as such, just which template to trigger.

Reply:
@jfsoul Exactly, the way we handle the failure doesn't change, so we have specific objects to trigger sending a specific failure email to an email address. We then look at the sent email and assess if it gets sent and the format is correct. It's fairly manual.